src/csv/builder.cr

```diff
@@ -35,15 +35,21 @@ require "csv"
 # 4,5,6,7,8
 # ```
 class CSV::Builder
+  enum Quoting
+    NONE # No quotes
```

Comment:
This documentation doesn't work, it needs to be before the constants to show up in `crystal doc`.